src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5117,6 +5117,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ExplicitReadOnlyAttr" xml:space="preserve">
     <value>Do not use 'System.Runtime.CompilerServices.IsReadOnlyAttribute'. This is reserved for compiler usage.</value>
   </data>
+  <data name="ERR_ExplicitIsByRefLikeAttr" xml:space="preserve">
```

Comment:
I remember the decision for some other errors (specifically language version errors) are based on telemetry. I don't think that's a concern here?

---
In reply to: [118795162](https://github.com/dotnet/roslyn/pull/19795#discussion_r118795162) [](ancestors = 118795162,118793105)